src/app/containers/Preferences/General/Aaxis.js

```diff
@@ -0,0 +1,59 @@
+import controller from 'app/lib/controller';
+import React, { useState } from 'react';
+import store from 'app/store';
+import ToggleSwitch from 'app/components/ToggleSwitch';
+import Tooltip from 'app/components/TooltipCustom/ToolTip';
+import Input from '../components/Input';
+import Fieldset from '../components/Fieldset';
+
+
+const Aaxis = () => {
+    const [addAAxisStepPerMM, setAddAAxisStepPerMM] = useState(false);
+    //TODO
```

Comment:
The value here needs to persist, so it should be saved and retrieved from the store